packages/module/src/PasswordInput/PasswordHiddenText.tsx

```diff
@@ -0,0 +1,43 @@
+import { Button, Flex, FlexItem, Content } from '@patternfly/react-core';
+import { EyeIcon, EyeSlashIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+export type PasswordHiddenTextProps = {
+  /** Read only password text to display */
+  password: string;
+  /** Test id to pass to the view password toggle button */
+  testId: string;
+};
+
+export const PasswordHiddenText: React.FunctionComponent<PasswordHiddenTextProps> = ({
+  password, 
+  testId 
+}: PasswordHiddenTextProps) => {
+  const [isHidden, setIsHidden] = React.useState(true);
+
+  const passwordText = isHidden ? Array(password.length).fill('\u25CF').join('') : password;
+
+  return (
+    <Flex
+      spaceItems={{ default: 'spaceItemsNone' }}
```

Comment:
Can this be `spaceItemsXs`? I think with a little space between the password and the button (especially when it is hovered or clicked) it would look better